### ISSUE TITLE
Panic when shutting down windowing by close button

### ISSUE THREAD

**Issue by cybersoulK** [orottier/web-audio-api-rs#387]
```rust
panicked at 'called `Result::unwrap()` on an `Err` value: "SendError(..)"', /Users/.../.cargo/registry/src/index.crates.io-6f17d22bba15001f/web-audio-api-0.36.1/src/context/concrete_base.rs:415:40

panicked at 'called `Result::unwrap()` on an `Err` value: "SendError(..)"', /Users/.../.cargo/registry/src/index.crates.io-6f17d22bba15001f/web-audio-api-0.36.1/src/context/concrete_base.rs:415:40

panicked at 'panic in a function that cannot unwind', library/core/src/panicking.rs:126:5
thread caused non-unwinding panic. aborting.
```

this happens when i shutdown the winit windowing by the close UI button, 
but it does not happen when i close the terminal by Ctrl + C

might wanna use?
```rust
self.send_control_msg(message).ok();
```

**Comment by orottier** [comment]
Thanks. We should indeed not use `unwrap` here.
It's probably best if `send_control_msg` absorbs all `SendErrors` and logs a warning and, if not already the case, puts the context in `Suspended` state.
I probably have time to fix this near the end of the week, but feel free to submit a PR yourself earlier.

**Comment by orottier** [comment]
I have prepared a fix in #394
